### ISSUE TITLE
TLS handshake error

### ISSUE THREAD

**Issue by philipzae** [IBBoard/cawbird#88]
I'm constantly getting this error in the UI or when I try to post a tweet - "Could not show tweet: Error performing TLS handshake: An unexpected TLS packet was received."

In the terminal, I saw this this error. I'm running cawbird 1.0.4 on Linux Mint 19.3
```
(cawbird:18680): cawbird-WARNING **: 09:07:20.409: ../../src/CbUtils.c(call_done_cb): 0x55f15ebc0c10, Error performing TLS handshake: An unexpected TLS packet was received.

(cawbird:18680): cawbird-WARNING **: 09:07:20.409: load_timeline_tweets_done: Error performing TLS handshake: An unexpected TLS packet was received. (rest-proxy-error-quark - 4)
```

![Screenshot_2020-02-03_08-27-38](https://user-images.githubusercontent.com/4412114/73626904-0a32d980-4664-11ea-9bc5-4c5af8b019f4.png)

**Comment by IBBoard** [comment]
Your distro needs to update GnuTLS to include a bug fix. I've done what I can to work around it in #9, but there's only so much that a work-around can achieve.

**Comment by philipzae** [comment]
is there a particular GnuTLS version that has this bug fix? Is the fix not available in ubuntu 18.04?

**Comment by IBBoard** [comment]
Details of the official fix from GnuTLS are in the other ticket: https://github.com/IBBoard/cawbird/issues/9#issuecomment-539156520
